docs/commands.rst

```diff
@@ -544,6 +544,35 @@ remote test machine.  You can run jobs on the local machine but it is not
 recommended since some tasks reboot the system. Hosts are tied to recipe IDs
 inside the job XML.
 
+Arguments for the client are as follows::
+
+    restraint --host <1=host> --job <job.xml> [--restraint-path </dir/restraintd>] [-v]
+
+Where:
+
+.. option:: --host <recipe_id>=[<user>@]<host>
+
+   Set host for a recipe with specific id.
```

Comment:
Based on this I still don't know why should I specify recipe_id.

Reply:
Sorry for delay.  I wanted to test it out before answering.  Updated code.

Reply:
See updated documentation.